include/picolibrary/microchip/megaavr/asynchronous_serial.h

```diff
@@ -78,6 +78,207 @@ enum class USART_Clock_Generator_Operating_Speed : std::uint8_t {
     DOUBLE = 0b1 << Peripheral::USART::Normal::UCSRA::Bit::U2X, ///< Double.
 };
 
+/**
+ * \brief Basic transmitter.
+ *
+ * \tparam Data_Type The integral type used to hold the data to be transmitted.
+ */
+template<typename Data_Type>
```

Comment:
Assert that `Data_Type` is `std::uint8_t` or `std::uint16_t` to ensure `load_transmit_buffer()` overload resolution will work properly.